packages/table-next/typings/react-table-config.d.ts

```diff
@@ -0,0 +1,133 @@
+//
+// NOTE: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/react-table
```

Comment:
If this is in DefinitelyTyped, why copy it here?

Reply:
it's a weird package. the instructions and reasonings are outlines in that link

https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/react-table#configuration-using-declaration-merging